code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -224,7 +224,7 @@
 	. += speed + slowdown + speed_modifier
 
 	if(frenzy_aura)
-		. -= (frenzy_aura * 0.05)
+		. -= (frenzy_aura * 0.12)
```

Comment:
If you could fiddle with this to be more in line with .1, .2, as the server ticks are in deci-seconds which means that .02 .04 won't ever get 'felt' by the player.